daemon/api.go

```diff
@@ -1779,6 +1786,26 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	}, nil)
 }
 
+func postUpdateRevisions(c *Command, r *http.Request, user *auth.UserState) Response {
+	uid, err := postCreateUserUcrednetGetUID(r.RemoteAddr)
```

Comment:
This will probably go away again, you suggested that we move boot-ok inside the daemon too and with that we do not need this endpoint anymore.